contracts/UFragmentsPolicy.sol

```diff
@@ -8,8 +8,8 @@ import "./lib/UInt256Lib.sol";
 import "./UFragments.sol";
 
 
-interface IMarketOracle {
-    function getPriceAnd24HourVolume() external returns (uint256, uint256);
+interface IOracle {
+    function getData() external returns (bool, uint256);
```

Comment:
Same comment as in the oracle diff. `(uint256, bool)` seems like a more natural ordering.

Reply:
👍